### PR TITLE
Redesign header and sidebar position

### PR DESCRIPTION
**Pull request by ianlaidlaw**
This pr redesigns the sidebar positioning and the header for LX.

Changes:
- The sidebar is always at the left side of the screen.
- The sidebar will fill 1/3 of the screen width on higher resolutions, until maxed out at 570px.
- The content will center itself inthe remaining 2/3 of the screen (or more when sidebar is maxed out)
- The header buttons have shifted, and the left content will animate as the sidebar slides out. The move separately, but this design was approved.

desktop_sidebar_2

desktop_sidebar_3

The Back to Content text will hide at lower resolutions

hide_header

### CODE REVIEW

File: components/d2l-sequence-viewer-iterator.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import '@polymer/polymer/polymer-legacy.js';
```
Comment (ianlaidlaw) [inline components/d2l-sequence-viewer-iterator.js:1]: This file is more or a less a copy + paste of the component that exists in sequences, with a rename and some minor style changes

File: d2l-sequence-viewer.js [hunk 2]
```diff
@@ -91,22 +90,24 @@ class D2LSequenceViewer extends mixinBehaviors([
 					inexplicably subtracting 12px from the height of the iframe,
 					and fixing that offset here will prevent a double scrollbar */
 					height: calc(100% + 12px);
+					/*Viewframe max width is 1170px, but viewer has 30px
+					inherent padding horizontally to account for.*/
+					max-width: calc(var(--viewer-max-width) + 2*var(--viewframe-horizontal-margin));
```
Comment (ianlaidlaw) [inline d2l-sequence-viewer.js:95]: gross. but i couldnt find a great way to handle this

File: d2l-sequence-viewer.js [hunk 3]
```diff
@@ -50,30 +52,27 @@ class D2LSequenceViewer extends mixinBehaviors([
...
 					position: relative;
 					overflow: hidden;
-					max-width: var(--viewer-max-width);
 					width: 100%;
-					margin: 30px auto 0 auto;
 				}
 				#sidebar-container {
 					height: 100%;
-					z-index: 2;
+					z-index: 1;
```
Comment (ianlaidlaw) [inline d2l-sequence-viewer.js:69]: This used to be 1 and at one point I changed it to 2 but it has no reason to be. This fixes the issue with the new content alert being behind the sidebar

image

Comment (sarahboyd) [reply d2l-sequence-viewer.js]: yay!!!

**Pull request merged**